backend/src/database.ts

```diff
@@ -227,14 +223,14 @@ export async function storeGame(game: GameState): Promise<void> {
 // Gets game from db and removes ELO ids before returning it.
```

Comment:
remove mention of ELO ids from this comment. (Also, see comments above about whether tokenIds need to be stripped).

Reply:
ok will do